src/System.Private.CoreLib/src/System/Reflection/MdImport.cs

```diff
@@ -554,7 +531,7 @@ public ConstArray GetMemberRefProps(int memberTokenRef)
             out int constructorToken,
             out ConstArray signature);
 
-        public void GetCustomAttributeProps(
+        public void GetCustomAttributeRecord(
```

Comment:
There does not seem to be a good reason for this rename anymore - it makes the style different from the other methods.

Reply:
Reverted